### DISCUSSION TITLE
I got banned but didn't receive email explaining why I got banned.

### DISCUSSION THREAD

**Discussion by teletvbisgotbanned** [community/community#116460] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Could I know why i got banned? My banned account is teletvbis.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @teletvbisgotbanned ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
